sys/shell/commands/shell_commands.c

```diff
@@ -200,144 +202,141 @@ extern int _cryptoauth(int argc, char **argv);
 extern int _bootloader_handler(int argc, char **argv);
 #endif
 
-const shell_command_t _shell_command_list[] = {
-    {"reboot", "Reboot the node", _reboot_handler},
-    {"version", "Prints current RIOT_VERSION", _version_handler},
+SHELL_COMMAND(reboot, "Reboot the node", _reboot_handler);
+SHELL_COMMAND(version, "Prints current RIOT_VERSION", _version_handler);
 #ifdef MODULE_USB_BOARD_RESET
-    {"bootloader", "Reboot to bootloader", _bootloader_handler},
+SHELL_COMMAND(bootloader, "Reboot to bootloader", _bootloader_handler);
 #endif
 #ifdef MODULE_CONFIG
-    {"id", "Gets or sets the node's id.", _id_handler},
+SHELL_COMMAND(id, "Gets or sets the node's id.", _id_handler);
 #endif
 #ifdef MODULE_HEAP_CMD
-    {"heap", "Prints heap statistics.", _heap_handler},
+SHELL_COMMAND(heap, "Prints heap statistics.", _heap_handler);
 #endif
 #ifdef MODULE_PERIPH_PM
-    { "pm", "interact with layered PM subsystem", _pm_handler },
+SHELL_COMMAND(pm, "interact with layered PM subsystem", _pm_handler);
 #endif
 #ifdef MODULE_PS
-    {"ps", "Prints information about running threads.", _ps_handler},
+SHELL_COMMAND(ps, "Prints information about running threads.", _ps_handler);
 #endif
 #ifdef MODULE_SHT1X
-    {"temp", "Prints measured temperature.", _get_temperature_handler},
-    {"hum", "Prints measured humidity.", _get_humidity_handler},
-    {"weather", "Prints measured humidity and temperature.", _get_weather_handler},
-    {"sht-config", "Get/set SHT10/11/15 sensor configuration.", _sht_config_handler},
+SHELL_COMMAND(temp, "Prints measured temperature.", _get_temperature_handler);
+SHELL_COMMAND(hum, "Prints measured humidity.", _get_humidity_handler);
+SHELL_COMMAND(weather, "Prints measured humidity and temperature.", _get_weather_handler);
+SHELL_COMMAND(sht_config, "Get/set SHT10/11/15 sensor configuration.", _sht_config_handler);
```

Comment:
Which then allows for
```suggestion
SHELL_COMMAND_CUSTOM(sht_config, "sht-config", "Get/set SHT10/11/15 sensor configuration.", _sht_config_handler);
```

here?